Assets/Scripts/Interaction Scripts/ControllerInput.cs

```diff
@@ -65,6 +66,32 @@ void Awake()
 		//  LEFT HAND GETTER METHODS  //
 		/******************************/
 
+		/// <summary>
+		/// Whether the left hand is currently touching an object.
+		/// </summary>
+        /// <returns>True if the left hand is currently touching an object.</returns>
+		public bool LeftIsTouching()
+		{
+			return (LeftController.GetComponent<VRTK_InteractTouch>().GetTouchedObject()) ? true : false;
+		}
+
+		/// <summary>
+		/// Whether the left hand is currently touching an object.
+		/// </summary>
+        /// <returns>True if the left hand is currently touching an object.</returns>
+		public bool LeftIsTouchingWaypoint()
+		{
+			GameObject touched = LeftController.GetComponent<VRTK_InteractTouch>().GetTouchedObject();
+			if (touched)
+			{
+				if (touched.tag == "waypoint")
```

Comment:
Can't we combine these two if statements (i.e. touched && (touched.tag == "waypoint"))?

Reply:
or just `return touched && (touched.tag == "waypoint")`

Reply:
good point, will do